hardware/raspberrypi/README.md

```diff
@@ -4,6 +4,8 @@ The hardware in the Raspberry Pi
 
 - [Schematics](schematics/README.md)
     - Schematics for the Raspberry Pi
+- [Datasheets](datasheets.md)
+    - Currently available datasheets for the Raspberry Pi range
```

Comment:
The schematics link doesn't say "Currently available schematics for the Raspberry Pi range", so maybe the text here should just say "Datasheets for the Raspberry Pi" ?

Reply:
Just trying to add a bit of variety!